packages/manager/core/generator/app/conditional-templates/onboarding/index.tsx.hbs

```diff
@@ -1,29 +1,39 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 import i18next from 'i18next';
+import {
+  useAuthentication,
+  useEnvironment,
+} from '@ovh-ux/manager-react-core-application';
 import { OnboardingLayout } from '../../components/layout-helpers/Onboarding';
 import { ManagerTile, ManagerTileType } from '../../components/ManagerTile';
 import onboardingImgSrc from './onboarding-img.png';
 import './index.scss';
 
+import constants from './constants';
```

Comment:
You should rename this file and use more meaningful variable names like `guides`